catalog/app/components/FileEditor/QuiltConfigEditor/QuiltSummarize/QuiltSummarize.tsx

```diff
@@ -0,0 +1,782 @@
+                onChangeValue('description', event.currentTarget.value)
+              }
+              value={file.description || ''}
+              fullWidth
+              className={classes.field}
+              size="small"
+            />
+
+            <M.FormControl className={classes.render}>
+              <M.FormLabel>Preview</M.FormLabel>
+              <M.FormControlLabel
+                className={classes.expanded}
+                control={
+                  <M.Checkbox
+                    checked={file.expand || false}
+                    onChange={(_e, expand) => onChangeValue('expand', expand)}
+                    size="small"
+                  />
+                }
+                labelPlacement="start"
+                label="Expand"
+                title="Whether preview is expanded by default or not"
+              />
+              <M.FormGroup>
+                {row.columns.length > 1 && (
+                  <M.TextField
+                    disabled={disabled}
+                    label="Width"
+                    name="width"
+                    onChange={(event) =>
+                      onChangeValue('width', event.currentTarget.value)
+                    }
+                    value={file.width || ''}
+                    fullWidth
+                    className={classes.field}
+                    size="small"
+                    helperText="Width in pixels or percents"
+                  />
+                )}
+
+                <M.FormControl className={classes.field} fullWidth size="small">
+                  <M.InputLabel>Renderer</M.InputLabel>
+                  <M.Select
+                    className={classes.select}
+                    value={file.type?.name || ''}
+                    onChange={(event) =>
+                      onChangeType('name', event.target.value as Summarize.TypeShorthand)
+                    }
+                  >
+                    <M.MenuItem value="">
+                      <i>Default</i>
+                    </M.MenuItem>
+                    {schema.definitions.typeShorthand.enum.map((type) => (
+                      <M.MenuItem key={type} value={type}>
+                        {type}
+                      </M.MenuItem>
+                    ))}
+                  </M.Select>
+                </M.FormControl>
+
+                {file.type && (
+                  <M.TextField
+                    disabled={disabled}
+                    label="Height"
+                    name="height"
+                    onChange={(event) =>
+                      onChangeType('style', { height: event.currentTarget.value })
+                    }
+                    value={file.type.style?.height || ''}
+                    fullWidth
+                    className={classes.field}
+                    size="small"
+                    placeholder="Ex., 1000px"
+                    helperText="Height as an absolute value (in `px`, `vh`, `em` etc.)"
```

Comment:
are `vh` and `em` absolute?

Reply:
They are not in CSS context, but I don't know how to better describe that one can enter number as ratio value in `flexGrow` or some concrete value with units.
Probably, "concrete"?